packages/cli/src/oclif/commands/convert.js

```diff
@@ -118,7 +118,7 @@ ConvertCommand.flags = buildFlags({
     })
   }
 });
-ConvertCommand.description = `Converts a Legacy Web Builder or Visual Builder app to a CLI app.
+ConvertCommand.description = `Convert a Legacy Web Builder or Visual Builder integration to a CLI integration.
```

Comment:
Might swap to `Convert a Legacy Web Builder app or Visual Builder integration to a CLI integration`, since "web builder app" is a known phrase.